appveyor.yml

```diff
@@ -32,8 +38,8 @@ install:
 build_script:
 # Need to convert from shallow to complete for Pkg.clone to work
   - IF EXIST .git\shallow (git fetch --unshallow)
-  - C:\projects\julia\bin\julia -e "versioninfo(); Pkg.init();
-      Pkg.clone(pwd(), \"Optimize\"); Pkg.build(\"Optimize\")"
+  - C:\projects\julia\bin\julia -e "VERSION >= v\"0.7\" && using InteractiveUtils, Pkg;
+      versioninfo(); Pkg.init(); Pkg.clone(pwd(), \"Optimize\"); Pkg.build(\"Optimize\")"
```

Comment:
No need for `Pkg.init()` in 0.7 and 1.0. No need for `InteractiveUtils` and `versioninfo()` either. It's printed by default.